tests/emulators.py

```diff
@@ -75,6 +75,7 @@ def __init__(
         port: Optional[int] = None,
         headless: bool = True,
         auto_interact: bool = True,
+        main_args: Sequence[str] = ("-m", "main"),
```

Comment:
dtto

Reply:
Changed in [7543c1d](https://github.com/trezor/trezor-firmware/pull/2857/commits/7543c1d620a59dd353638d7254e3832d35c8ea12)